stackstorm_withitems/actions/with_items.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: "with_items"
+runner_type: "python-script"
+description: "faster with items;"
+enabled: true
+entry_point: "with_items.py"
+parameters:
+  action:
+    type: string
+    required: true
+  parameters:
+    type: array
+    items:
+      type: object
+    description: "list of objects for parameters"
+  paging_limit:
+    type: integer
+    default: 5
+  sleep_time:
+    type: integer
+    default: 25
+  result:
+    type: string
+    description: "jinja string to {_{ _.result.xxx }_} portion of result to return"
```

Comment:
I don't understand what this is trying to tell me, but I haven't had my ☕ yet today.

Reply:
you can specify a part of the json to return.  see the unit tests for examples.

Reply:
Okay, that explains that, but this parameter should have a better description so people don't have to dig through the unit tests for examples. You should be able to give examples, with expected output, right in the description.